go/client/fork_server.go

```diff
@@ -140,6 +140,12 @@ func makeServerCommandLine(cl libkb.CommandLine, forkType keybase1.ForkType) (ar
 		}
 	}
 
+	// If there is no explicit log file add one when autoforking.
+	// otherwise it was added in the previous block already.
+	if cl.GetLogFile() == "" {
+		args = append(args, "--log-file", G.Env.GetDefaultLogFile())
```

Comment:
We're avoiding all naked `G` references going forward and are instead plumbing through `g` from above. Shouldn't be too hard since `spawnServer` has it.